src/overview/__tests__/overviewActions.test.ts

```diff
@@ -1,39 +1,43 @@
+import { MockStore } from 'redux-mock-store'
+
+import { generateMockStore } from '__mock__/mockStore.mock'
 import {
   getOverviewData,
   OVERVIEW_RECEIVED_ACTION,
   overviewReceivedAction
 } from 'overview/overviewActions'
-import { initialState } from 'overview/overviewReducer'
-import configureMockStore from 'redux-mock-store'
-import thunk from 'redux-thunk'
 
-import { mockApi } from './__mock__/mockApi'
-import { mockAaplOverviewData } from './__mock__/mockData'
+import { mockApi } from '__mock__/api.mock'
+import { mockGlobalState } from '__mock__/globalstate.mock'
 
 describe('overview actions', () => {
+  let store: MockStore<{}>
+  const overviewData = mockGlobalState.overview
+
+  beforeEach(() => {
+    store = generateMockStore(mockGlobalState , mockApi)
+    store.clearActions()
+  })
+  
   it('should create action update overview data', () => {
     const expectedAction = {
```

Comment:
could maybe create a variable so the object could be used later in the last test as well?